src/components/question-wrapper.tsx

```diff
@@ -159,9 +151,17 @@ export default class QuestionWrapper extends React.Component<IProps, IState> {
     );
   }
 
+  private renderDistractorToggle() {
```

Comment:
This seemed like overkill to me, but I couldn't find a way to otherwise shorten line 112 so the linter wouldn't complain. And spreading the code over multiple lines always seemed to cause the Activity Player to crash.